extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -142,7 +142,7 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  documentInactiveCallback() {
+  unlayoutCallback() {
```

Comment:
@mkhatib This part is already taken care of by `viewportCallback`. Basically, the carousel stops autoplay when out of viewport. And `pauseCallback` sends "out of viewport" signal to the element.